packages/apollo-collaboration-server/src/fileHandling/fileHandling.service.ts

```diff
@@ -503,4 +508,107 @@ export class FileHandlingService {
     this.logger.debug('GFF3 is not loaded in cache!')
     return false
   }
+
+  /**
+   * This method check that each line has unique id. If not it creates one for each line and overwrites the orignal file
+   * @param filenameWithPath - filename with path
+   */
+  async checkGFF3uniqueKey(filenameWithPath: string) {
+    const stringOfGFF3 = await fs.readFile(filenameWithPath, {
+      encoding: 'utf8',
+      flag: 'r',
+    })
+    this.logger.verbose(`Data read from file=${stringOfGFF3}`)
+
+    const arrayOfThings = gff.parseStringSync(stringOfGFF3, {
+      parseAll: true,
+    })
```

Comment:
I was wrong when we were talking earlier, this is right. We want `parseAll` so that the sequence, etc. ends up back in the file.

Reply:
ok